channels/ca.m3u

```diff
@@ -99,6 +101,12 @@ https://d7z3qjdsxbwoq.cloudfront.net/groupa/live/f9809cea-1e07-47cd-a94d-2ddd3e1
 http://159.69.58.154/cpmtv/cpmtv.m3u8
 #EXTINF:-1 tvg-id="CTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/nfcjHAW.png" group-title="",CTV (720p)
 https://pe-fa-lp02a.9c9media.com/live/News1Digi/p/hls/00000201/38ef78f479b07aa0/index/0c6a10a2/live/stream/h264/v1/3500000/manifest.m3u8
+#EXTINF:-1 tvg-id="DesheBidesheTV.ca" tvg-country="CA" tvg-language="Bengali" tvg-logo="https://www.deshebideshe.tv/assets/importent_images/footer-logo.png" group-title="General",DesheBideshe TV (720p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/deshebideshe.stream/playlist.m3u8
```

Comment:
Displays banner. No live streaming.

Reply:
It works fine for me, even through a VPN.
![vlcsnap-2021-10-02-22h39m10s936](https://user-images.githubusercontent.com/7253922/135729825-973393c1-5898-4778-9300-903f15d843f0.png)

Reply:
Yes, it's streaming here now. May have been a break, and that was a placeholder.

Reply:
Thanks for your help!